pkg/auth/tokenexchange/exchange.go

```diff
@@ -158,8 +158,8 @@ func (c clientAuthentication) String() string {
 		c.ClientID, clientSecret)
 }
 
-// Config holds the configuration for token exchange.
-type Config struct {
+// ExchangeConfig holds the configuration for token exchange.
+type ExchangeConfig struct {
```

Comment:
i believe that this was changed to avoid naming collision in the Config middleware?